src/main/java/edu/greenblitz/robotName/RobotConstants.java

```diff
@@ -4,7 +4,7 @@
 import edu.wpi.first.wpilibj.PowerDistribution;
 
 public class RobotConstants {
-  public static final Robot.RobotType ROBOT_TYPE = Robot.RobotType.SIMULATION;
+  public static final Robot.RobotType ROBOT_TYPE = Robot.RobotType.ROBOT_NAME;
```

Comment:
we need a way tomake this configurable, I see a match that we will not work because the robot is in `SIMULATION`

Reply:
but not in this PR, we should do a seperate branch